ros2/param.bzl

```diff
@@ -0,0 +1,25 @@
+""" Implements a macro for setting up a ROS2 param app.
```

Comment:
What's the motivation for ros2_param macro? I though ros2_param binary (defined in https://github.com/mvukov/rules_ros2/blob/main/ros2/BUILD.bazel#L32) is sufficient. BTW, you can alias it in your workspace as https://github.com/mvukov/rules_ros2/blob/main/examples/BUILD.bazel#L2.

Reply:
you're right!